chainer/testing/__init__.py

```diff
@@ -1,4 +1,4 @@
-import nose
+import pytest
```

Comment:
This line will cause `ImportError` when importing `chainer.testing` in the environment without `pytest`. (pytest is no longer a requirement for normal chainer installation).
How about importing `pytest` in `run_module` function?

Reply:
Fixed as suggested!